compilers/concrete-compiler/compiler/lib/Bindings/Python/concrete/compiler/compilation_feedback.py

```diff
@@ -135,9 +135,13 @@ def count_per_tag(self, *, operations: Set[PrimitiveOperation]) -> Dict[str, int
             if statistic.operation not in operations:
                 continue
 
-            tag, _, _ = re.match(REGEX_LOCATION, statistic.location).groups()
-            # remove the @
-            tag = tag[1:] if tag else ""
+            match = REGEX_LOCATION.match(statistic.location)
```

Comment:
Could be a separate function tag_from_location as it's repeated.

Reply:
Done.